lib/swagger/diff/specification.rb

```diff
@@ -84,9 +92,16 @@ def parsed_to_hash(parsed)
       # parameter definitions (i.e., all parameters, including nested
       # parameters, are included in a single set).
       def refs(ref, prefix = '')
+        defs = if ref[0..12] == '#/parameters/'
+                 @parsed['parameters']
+               elsif ref[0..11] == '#/responses/'
+                 @parsed['responses']
+               else
+                 @parsed['definitions']
```

Comment:
I would probably explicitly check for `'#/definitions/' to and throw an error if none of the cases are met.